elan-init.sh

```diff
@@ -98,7 +98,7 @@ main() {
     fi
 
     ensure mkdir -p "$_dir"
-    local _latest=$(ensure curl -sSf "$ELAN_UPDATE_ROOT/latest" | cut -d'"' -f2 | awk -F/ '{print $NF}')
+    local _latest=$(ensure curl -Ls -o /dev/null -w %{url_effective} "$ELAN_UPDATE_ROOT/latest" | cut -d'"' -f2 | awk -F/ '{print $NF}')
```

Comment:
It seems like just adding `L` is enough to fix this? The `-w` flag seems pointless to me in the presense of `-o /dev/null`, and it's not clear to me that dropping `S` and `f` is beneficial.

(I'm using https://explainshell.com/explain?cmd=curl+-LsSf+-o+%2Fdev%2Fnull+-w+%25%7Burl_effective%7D  for reference)

Reply:
> It seems like just adding `L` is enough to fix this?

It doesn't for me. We want the redirection URL, not its content.

Reply:
> and it's not clear to me that dropping `S` and `f` is beneficial.

I agree, though it's not terribly helpful without `-o pipefail`